src/main/java/seedu/address/logic/commands/AddItemTagCommand.java

```diff
@@ -29,9 +29,10 @@
             + "Example: " + COMMAND_WORD + PREFIX_ITEM_NAME + "Iron "
             + PREFIX_ITEM_TAG + "delicious, tuturu";
 
-    public static final String MESSAGE_ITEM_NOT_PROVIDED = "Item was not found in the item list.";
+    public static final String MESSAGE_ITEM_NOT_PROVIDED = "Item name was not provided.";
     public static final String MESSAGE_ITEM_NOT_FOUND = "Item was not found in the item list.";
     public static final String MESSAGE_TAG_NOT_ADDED = "No new tag is added.";
```

Comment:
from below code it looks like this will only show iff the user input is exactly the current tags. should it also show if it's a subset?